src/EditorFeatures/Test/CodeGeneration/CodeGenerationTests.VisualBasic.cs

```diff
@@ -1,21 +1,17 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeGeneration;
-using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Editing;
-using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.CodeAnalysis.VisualBasic;
 using Microsoft.CodeAnalysis.VisualBasic.Syntax;
 using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
```

Comment:
Maybe the "sort system first" style should be noted [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) as well then, and be enforced by CodeFormatter?